task3.js

```diff
@@ -0,0 +1,77 @@
+const taskOneButton = document.getElementsByClassName('task-one-button')[0]
+// const firstInput = document.querySelector('#first-input')
+// const secondInput = document.querySelector('#second-input')
+
+const downloads = [
+  {
+    id: 1,
+    title: 'Recipe',
+    status: 'Done',
+  },
+  {
+    id: 2,
+    title: 'Workouts',
+    status: 'Pending',
+  },
+  {
+    id: 3,
+    title: 'Passwords',
+    status: 'Pending',
+  },
+  {
+    id: 4,
+    title: 'To Do 2021',
+    status: 'Pending',
+  },
+  {
+    id: 5,
+    title: 'Books',
+    status: 'Failed',
+  },
+]
+
+function changeItemProperty() {
+  const currentInterval = setInterval(main, 5000)
+  function main() {
+    console.log('Check started')
+    const filtered = downloads.filter((e) => e.status === 'Pending')
+    if (filtered.length) {
+      for (let i = 0; i < downloads.length; i++) {
```

Comment:
try to use array functions like array.find, .map etc

Reply:
I've used array.find to dinf element with status 'Pending'